src/cio_websocket.c

```diff
@@ -259,6 +259,40 @@ static void handle_close_frame(struct cio_websocket *ws, uint8_t *data, uint64_t
 
 static void get_header(struct cio_buffered_stream *bs, void *handler_context, enum cio_error err, struct cio_read_buffer *buffer);
 
+static void restart_reading(struct cio_buffered_stream *bs, void *handler_context, const struct cio_write_buffer *buffer, enum cio_error err)
+{
+	(void)bs;
+	(void)buffer;
+	struct cio_websocket *ws = (struct cio_websocket *)handler_context;
+	if (err != CIO_SUCCESS) {
+		// TODO: handle_error(s, WS_CLOSE_PROTOCOL_ERROR);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this block of commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=c%3ACommentedCode)
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this "TODO" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=c%3AS1135)